### ISSUE TITLE
BUG - sev-3 - All - Center-align icon with description on deeplink and large nav buttons, when font resized larger

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#8309]
## What happened?

When a user resizes the font on their device to much larger, per VQA, "for the icon alignment. I'd expect it to center align with the description"

Here's the current behavior (home deeplink button 'top' aligned; category screen large nav button 'bottom' aligned):
<img width="385" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBN0EzQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--ec4e29e6da85f0a05ad5a58c48435e3bd5863d6a/image.png" alt="image.png" />

<img width="385" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBN0UzQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--8b6b2dcaf56cf1a9c6cee9ef94bae2a498d6711e/image.png" alt="image.png" />

## Specs:

- Device:
- OS:
- User Account:
- Accessibility State:

## Desired behavior

Icon should be center-aligned with description text.

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: High Low 
- Frequency: High Low 

- 3 - Low
- 2 - High
- 1 - Critical

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))

**Comment by TKDickson** [comment]
@theodur I'm seeing the same root problem on the build I made from your branch. Did something go awry with the build process, or is it a result of the text wrapping?

<img width="368" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBM0p6QkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--5ba1eba7f6f92a0a7c70a013589c117643a8b292/image.png" alt="image.png" />

<img width="368" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMjl6QkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--6e32c0d546a702ec1d4cf484a9091dc64c6fff87/image.png" alt="image.png" />

**Comment by theodur** [comment]
@TKDickson Strange, even when amping up the text size to make it wrap, I'm still seeing them aligned

<img width="549" alt="image" src="https://github.com/department-of-veterans-affairs/va-mobile-app/assets/9042882/3e83f314-97d1-4d4e-b22e-b9c865561c60">

Are you noticing the same thing on Android too?

**Comment by TKDickson** [comment]
Weird. And no - on my Android devices even at the largest/largest for text & display, the text doesn't wrap. So it looks fine (and so did iOS on the non-text-wrapping font sizes).

Let me try.... building it again? We'll see what happens.

**Comment by TKDickson** [comment]
This is working on my new build. Great! I'll take it.

**Comment by TKDickson** [comment]
Closing as fixed. Merged to feature branch.
